ivadomed/config/config.json

```diff
@@ -8,6 +8,13 @@
         "object_detection_path": null,
         "safety_factor": [1.0, 1.0, 1.0]
     },
+    "wandb": {
+        "wandb_api_key": "",
+        "project_name": "ivado-wandb-testing",
+        "group_name": "temp",
+        "run_name": "run-1",
+        "log_grads_every": 100
```

Comment:
Here's how the "wandb" dict works based on the latest tests:

1. **When "wandb_api_key" is included in the dict and the API key authentication is successful:** The training proceeds without issues, metrics are logged on **both** WandB and Tensorboard. 
2. **When "wandb_api_key" is included in the dict and the API key authentication is unsuccessful:** WandB tracking is disabled, the training proceeds (does not stop) by just logging metrics on Tensorboard. A print statement notifies that WandB is disabled.
3. **When "wandb_api_key" is NOT included in the dict:** Error thrown and the training stops. This key is necessary.
4. **When "wandb_api_key" is included but is an empty string:** For e.g. as `"wandb_api_key": ""`, the training proceeds with **both** WandB and Tensorboard enabled **only if** there was a previous successful authentication. This is because once wandb has successfully authenticated, it stores the api key locally so that every time the user would not need to authenticate it. So, this also means that ivadomed's users also need to authenticate it only once and leave the field empty thereafter.

I think this covers all the possible cases, so if there's no other issue then this PR can be merged!

Reply:
Cool. I think as long as the default template has been updated, this is not an issue. The `wandb` key probably should be updated in all the config JSONs?

Reply:
Yes, I thought of creating a separate PR just for updating the docs and config files!

Reply:
@uzaymacar I just checked for the situation when the parent "wandb" key in the config file is not included. The training runs as usual. See my edited [comment above](https://github.com/ivadomed/ivadomed/pull/1069#discussion_r846307049)! Thanks again for bringing up this edge-case which I had missed! 🙌